documentation/osl_authorized_keys.md

```diff
@@ -0,0 +1,50 @@
+# osl_authorized_keys
+
+## Actions
+
+- `Add`: Adds a key from selected file.
+- `Remove`: Removes a key from selected file.
+
+## Properties
+
+| Property     | Type   | Default          | Required | Description                                                 |
+|--------------|--------|------------------|----------|-------------------------------------------------------------|
+| `key`        | String | None             | yes      | The ssh public key to be added (name property)              |
+| `user`       | String | None             | yes      | The user who owns the `authorized_keys` file at `dir_path`  |
```

Comment:
Name property should be in the Defaults column:
If there is no default, leave it blank:
```suggestion
| `key`        | String | Resource name    | yes      | The ssh public key to be added              |
| `user`       | String |                  | yes      | The user who owns the `authorized_keys` file at `dir_path`  |
```